mlflow/utils/environment.py

```diff
@@ -23,7 +23,7 @@ def _mlflow_conda_env(path=None, additional_conda_deps=None, additional_pip_deps
     :return: `None` if `path` is specified. Otherwise, the a dictionary representation of the
              Conda environment.
     """
-    env = yaml.load(_conda_header)
+    env = yaml.load(_conda_header, Loader=yaml.SafeLoader)
```

Comment:
Can you just change it to call yaml.safe_load instead? That should be a shorter way of saying the same thing and it's what we use in other places.

Reply:
Done. thx